ngraph/src/ngraph/frontend/onnx_import/op/upsample.cpp

```diff
@@ -0,0 +1,183 @@
+//*****************************************************************************
+// Copyright 2017-2020 Intel Corporation
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//*****************************************************************************
+
+#include <memory>
+
+#include "default_opset.hpp"
+#include "exceptions.hpp"
+#include "upsample.hpp"
+
+namespace ngraph
+{
+    namespace onnx_import
+    {
+        namespace op
+        {
+            namespace
+            {
+                bool check_mode_support(const onnx_import::Node& node, const std::string mode)
+                {
+                    std::unordered_set<std::string> supported_modes = {"nearest", "linear"};
```

Comment:
```suggestion
                    static const std::unordered_set<std::string> supported_modes = {"nearest", "linear"};
```

Reply:
I am ok with `const`, but unconvinced that we really want `static` here.